webapp/src/components/github_repo_selector/github_repo_selector.jsx

```diff
@@ -4,23 +4,24 @@
 import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 
-import Client from 'client';
 import ReactSelectSetting from 'components/react_select_setting';
 
 const initialState = {
     invalid: false,
     error: null,
-    repoOptions: [],
 };
 
 export default class GithubRepoSelector extends PureComponent {
     static propTypes = {
-        required: PropTypes.bool,
+        yourRepos: PropTypes.any,
```

Comment:
From the definition of the reducer, it looks like this will always be present, and always be an array. Is this if block needed?
```suggestion
        yourRepos: PropTypes.array.isRequired,
```

Reply:
You are correct.  nice catch.